test/dotnet-new.Tests/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "1.0.0-*",
+  "dependencies": {
+    "Microsoft.NETCore.App": {
+      "type": "platform",
+      "version": "1.0.0-rc2-*"
+    },
+    "Microsoft.DotNet.Tools.Tests.Utilities": {
+      "target": "project"
+    },
+    "Newtonsoft.Json": "7.0.1",
+    "dotnet": {
+      "target": "project"
+    },
+    "xunit": "2.1.0",
+    "dotnet-test-xunit": "1.0.0-dev-140469-38"
+  },
+  "frameworks": {
+    "netcoreapp1.0": {
+      "imports": [
+        "netstandardapp1.5",
```

Comment:
Do we still have a limitation somewhere requiring this to be netstandardapp? When we move to 1.6, it won't be there.